### PR TITLE
getfacl.md: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/getfacl.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# getfacl
+
+> Get file access control lists.
+
+- Display the file access control list:
+
+`getfacl -a {{path/to/file_or_folder}}`
```
Comment (agnivade) [inline pages/linux/getfacl.md]: This seems to give the same result as without `-a`. Let's remove it in that case.

Comment (jsonbruce) [reply pages/linux/getfacl.md]: Yes, you're right. The default operation of `getfacl` is `getfacl -a`

Comment (jsonbruce) [reply pages/linux/getfacl.md]: But I don't think `getfacl -PR` will be useful for most situations. The output format of `getfacl -PR` is not as good as `ls -al`.
So I think `getfacl` is more suit for single file.

Comment (agnivade) [reply pages/linux/getfacl.md]: Understood. Let's do the first part then.

**Pull request merged**